packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -27,12 +28,17 @@ function filenameFromIndex(index: number): string {
     return index === 0 ? "" : index.toString(); // support old tools...
 }
 
+let currSeq: number;
+
 async function* loadAllSequencedMessages(
     documentService?: IDocumentService,
     dir?: string,
-    files?: string[]) {
-    let lastSeq = 0;
+    files?: string[],
+    sequenceNumber: number = 0) {
```

Comment:
When you are never passing this seq number in this function, then why are you taking this as an argument?